frontend/src/static/css/style.scss

```diff
@@ -299,6 +300,18 @@ header {
       }
     }
   }
+
+  .commit-message {
```

Comment:
Seems we got two div with same `.commit-message` now? Why it is necessary?

Reply:
removed